### PR TITLE
aqrdev: rework internal structure

### PR DESCRIPTION
**Pull request by jecluis**
This patchset focus on reworking `aqrdev`'s internal organization, splitting off auxiliary functions into a separate module to then be consumed by upcoming work, reducing the amount of code in the one single file we had.

Signed-off-by: Joao Eduardo Luis \<joao@suse.com>

### CODE REVIEW

**Review by mgfritch — APPROVED** [review]

**Comment by AvengerMoJo** [comment]
Just to say in my PR #452 the concept of "Provider" for vagrant is added and it will be still missing in the new structure. If the user using vagrant not only using libvirt then it may create some issue.

**Comment by jecluis** [comment]
> Just to say in my PR #452 the concept of "Provider" for vagrant is added and it will be still missing in the new structure. If the user using vagrant not only using libvirt then it may create some issue.

No worries. I'll rebase your PR and fix the missing pieces.

File: tools/aqrdev [hunk 1]
```diff
@@ -852,8 +852,14 @@ def cmd_stop(ctx: AppCtx, name: str) -> None:
 @app.command("shell")
 @click.argument("name", required=True, type=str)
 @click.option("-n", "--node", required=False, type=str)
+@click.option("-c", "--command", required=False, type=str)
```
Comment (jhesketh) [inline tools/aqrdev:363]: (nit) We should pass in a help string

**Review by jhesketh — APPROVED** [review]
Looks good, although it is difficult to review such a large change so I am mostly relying on the successful wasser build and the fact that this is a refactor rather than all new code.

File: tools/aqrdev (hunk 1 above)
Comment (jecluis) [reply tools/aqrdev:363]: Agreed. This should be done for pretty much all of the commands :\

**Pull request merged**